api/app/Services/Api/Vanilla/Api/ApiInterface.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+namespace App\Services\Api\Vanilla\Api;
+
+interface ApiInterface
+{
+}
```

Comment:
@xiphiaz I've removed this interface. I initially added it to define how to handle paged results in different API group classes. Though after implementing further handling paging in this way was not required.